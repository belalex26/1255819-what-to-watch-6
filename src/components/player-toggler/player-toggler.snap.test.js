
import React from 'react';
import {Provider} from 'react-redux';
import {render} from '@testing-library/react';
import {Router} from 'react-router-dom';
import configureStore from 'redux-mock-store';
import {createMemoryHistory} from 'history';

import PlayerToggler from './player-toggler';

const mockStore = configureStore({});

it(`PlayerToggler renders correctly`, () => {
  const history = createMemoryHistory();
  const togglerProgress = 0;

  const {container} = render(
      <Provider store={mockStore({})}>
        <Router history={history}>
          <PlayerToggler
            togglerProgress={togglerProgress}
          />
        </Router>
      </Provider>
  );

  expect(container).toMatchSnapshot();
});

