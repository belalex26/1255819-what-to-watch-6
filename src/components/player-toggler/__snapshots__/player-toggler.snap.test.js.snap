// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlayerToggler renders correctly 1`] = `
<div>
  <div
    class="player__time"
  >
    <progress
      class="player__progress"
      max="100"
      value="0"
    />
    <div
      class="player__toggler"
      style="left: 0%;"
    >
      Toggler
    </div>
  </div>
</div>
`;
